### PR TITLE
Feat: most tackles and min tackles methods_VS

### PR DESCRIPTION
**Pull request by vsanantone**
Completed Most and Min Tackle methods by season. Difference of one enumerable.

### CODE REVIEW

File: lib/season.rb [hunk 1]
```diff
@@ -1,10 +1,70 @@
...
   def initialize(game_team_data, game_data, team_data)
+
     @game_team_data = game_team_data
     @game_data = game_data
     @team_data = team_data
+    @games_data = games_data
   end
 
+
+  def most_tackles(season)
```
Comment (EricBelongea) [inline lib/season.rb:11]: We could break these into helper methods or branch them off into modules.

Stuff for tomorrows refactoring

**Review by zkevkev — APPROVED** [review]
Thank you for these methods, they look fantastic! I think we'll have an opportunity to break this up into helper methods when we have our refactoring party tomorrow.

File: spec/season_spec.rb
Comment (EricBelongea) [inline spec/season_spec.rb:1]: This might create local merge conflicts for peeps

**Review by billyburch — APPROVED** [review]

**Review by EricBelongea — APPROVED** [review]
Good work group merge conflicting on video chat.

**Pull request merged**